app/src/main/res/values/strings.xml

```diff
@@ -14,6 +14,9 @@
     <string name="settings_backup_location_none">None</string>
     <string name="settings_backup_location_internal">Internal storage</string>
     <string name="settings_backup_last_backup_never">Never</string>
+    <string name="settings_backup_dialog_title">Really turn off backup?</string>
+    <string name="settings_backup_dialog_message">Turning off backup will... [prevent any currently-accessible backups from being restorable in the future] (fill this out later)</string>
```

Comment:
So it's either this, or once #477 is merged, we can say something about a brand new backup being created when re-enabled / mention the new restore set.

Ideally we don't block the dialog on the other change for too long.